src/easier68k/core/enum/ea_mode.py

```diff
@@ -0,0 +1,145 @@
+"""
+Represents an effective addressing mode
+and methods associated with it
+"""
+from src.easier68k.core.util.parsing import parse_literal
```

Comment:
This should use a relative import (if that's what it's called)

See here for an example: https://github.com/Chris-Johnston/Easier68k/blob/master/src/easier68k/simulator/memory.py#L7

Reply:
going to fix in another PR